src/svgs/EyeOpenedIcon.tsx

```diff
@@ -1,35 +1,15 @@
 import React from "react"
 import { Icon, IconProps } from "./Icon"
 
-interface OpenEyeIconProps extends IconProps {
```

Comment:
This file name should be preserved, and if a rename is necessary it can exist as a duplicate alongside the deprecated old file until a follow-up cleanup PR, at which point it can be fully deprecated / removed.